### DISCUSSION TITLE
Developer tenant Teams for Education

### DISCUSSION THREAD

**Discussion by IlonaHamanova** [microsoft/hack-together-teams#49] [Q&A]
Hello. I am teacher at technical high school, teaching computer sience and programing. I would like to try to develop apps for MS Teams with some of my students. But the developer tenant is only buisness and we want to develop for Teams for education. Our school has TfE, but we cant develop on it because of safety isues. So we need some other tenant of TfE for development. How can we get it? Thans for answer. Tried to ask on many places, but still have no answer. :-/

**Comment by waldekmastykarz** [comment]
That's a very good question. Let me ask around and I'll get back to you as soon as I know more

**Reply by IlonaHamanova** [reply]
@waldekmastykarz  Would be great, I hope you can help me somehow :-)

**Reply by waldekmastykarz** [reply]
Unfortunately, it seems like at the moment we don't have anything to offer. I'm sorry I don't have a better answer for you.

**Comment by svrooij** [comment]
@IlonaHamanova Can I reach you somewhere private? I can provide you with an education tenant that lasts 30 days, will that help?

It's called the [CDX](https://cdx.transform.microsoft.com/) environment, where some people (Microsoft partners, MVP's and others) can request a tenant for testing that is pre-filled with some users and some Teams.

**Reply by IlonaHamanova** [reply]
Hi, svorooij. Thanks for your offer, but I would like to use it longer than 30 days.
